lansa-vmss-windows-autoscale-sql-database/azuredeploy.json

```diff
@@ -0,0 +1,829 @@
+                      "[concat('https://', variables('uniqueStringArray')[2], variables('newStorageAccountSuffix'), '.blob.core.windows.net/', variables('vhdContainerName'))]",
+                      "[concat('https://', variables('uniqueStringArray')[3], variables('newStorageAccountSuffix'), '.blob.core.windows.net/', variables('vhdContainerName'))]",
+                      "[concat('https://', variables('uniqueStringArray')[4], variables('newStorageAccountSuffix'), '.blob.core.windows.net/', variables('vhdContainerName'))]"
+                    ],
+                     "name": "[variables('osDiskName')]",
+                     "caching": "ReadOnly",
+                     "createOption": "FromImage"
+                  },
+                  "imageReference": "[variables('imageReference')]"
+               },
+               "osProfile": {
+                  "computerNamePrefix": "[variables('dbvmssName')]",
+                  "adminUsername": "[parameters('adminUsername')]",
+                  "adminPassword": "[parameters('adminPassword')]",
+                  "windowsConfiguration": {
+                     "provisionVMAgent": "true"
+                  }
+               },
+               "networkProfile": {
+                  "networkInterfaceConfigurations": [
+                     {
+                        "name": "[concat('db', variables('nicName'))]",
+                        "properties": {
+                           "primary": "true",
+                           "ipConfigurations": [
+                              {
+                                 "name": "[concat( 'db',variables('ipConfigName'))]",
+                                 "properties": {
+                                    "subnet": {
+                                       "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/virtualNetworks/', variables('virtualNetworkName'), '/subnets/', variables('subnetName'))]"
+                                    },
+                                    "loadBalancerBackendAddressPools": [
+                                       {
+                                          "id": "[concat('/subscriptions/', subscription().subscriptionId,'/resourceGroups/', resourceGroup().name, '/providers/Microsoft.Network/loadBalancers/', variables('dbloadBalancerName'), '/backendAddressPools/', variables('dbbePoolName'))]"
+                                       }
+                                    ]
+                                 }
+                              }
+                           ]
+                        }
+                     }
+                  ]
+               },
+               "extensionProfile": {
+                  "extensions": [
+                     {
+                        "name": "FirstInstall",
+                        "properties": {
+                           "publisher": "Microsoft.Compute",
```

Comment:
This code:
```
                           "publisher": "Microsoft.Azure.Extensions",
                           "type": "CustomScript",
                           "typeHandlerVersion": "2.0",

```
Gets this error:
```
        "message": "Handler 'Microsoft.Azure.Extensions.CustomScript' has reported failure for VM Extension 'FirstInstall' with terminal error 
code '1007' and error message: 'Install failed for plugin (name: Microsoft.Azure.Extensions.CustomScript, version 2.0.2) with exception The 
specified executable is not a valid application for this OS platform.'"
```
Is this a Linux extension? I'm using Windows.

For Windows this is what I find: [Windows Custom Script Doc](https://azure.microsoft.com/en-us/documentation/articles/virtual-machines-windows-extensions-customscript/)